easybuild/tools/config.py

```diff
@@ -517,6 +520,32 @@ def get_log_filename(name, version, add_salt=False):
     return filepath
 
 
+def find_last_log(curlog):
+    """Find location to last log file that is still available."""
+    variables = ConfigurationVariables()
+    log_dir = get_build_log_path()
+    if variables['tmp_logdir'] is None:
+        # take info account that last part of default temporary logdir is random, if --tmp-logdir is not specified
+        log_dir = os.path.join(os.path.dirname(log_dir), '*')
+
+    glob_pattern = os.path.join(log_dir, 'easybuild*.log')  # see init_logging
+    _log.info("Looking for log files that match filename pattern '%s'...", glob_pattern)
+
+    try:
+        # sorted by modification time, most recent last
+        sorted_paths = sorted(glob.glob(glob_pattern), key=lambda f: os.stat(f).st_mtime)
+    except OSError as err:
+        raise EasyBuildError("Failed to locate log files matching '%s': %s", glob_pattern, err)
+
+    # log of current session is typically listed last, should be taken into account
+    res = sorted_paths[-1]
+    if os.path.samefile(res, curlog):
+        res = sorted_paths[-2]
```

Comment:
You could also check if the current user is the owner of the file...there could be a corner case where two EB-ers are using the same system at the same time.